README.md

```diff
@@ -4,12 +4,19 @@
 
 cuML is a suite of libraries that implement machine learning algorithms and mathematical primitives functions that share compatible APIs with other [RAPIDS](https://rapids.ai/) projects.
 
-cuML enables data scientists, researchers, and software engineers to run traditional tabular ML tasks on GPUs without going into the details of CUDA programming.
+cuML enables data scientists, researchers, and software engineers to run
+traditional tabular ML tasks on GPUs without going into the details of CUDA
+programming. In most cases, cuML's Python API matches the API from
+[scikit-learn](https://scikit-learn.org).
+
+For large datasets, these GPU-based implementations can complete 10-50x faster
+than their CPU equivalents. For details on performance, see the [cuML Benchmarks
+Notebook](https://github.com/rapidsai/notebooks-extended/blob/master/intermediate_notebooks/benchmarks/cuml_benchmarks.ipynb).
 
 As an example, the following Python snippet loads input and computes DBSCAN clusters, all on GPU:
 ```python
 import cudf
-from cuml import DBSCAN
+from cuml.cluster import DBSCAN
```

Comment:
```suggestion
from cuml import DBSCAN
```

The .cluster is not needed, so we want to change the default example here?

Reply:
This is more of a question than a change request

Reply:
To me, it was nice to match what sklearn does, since you would do from sklearn.cluster import DBSCAN and not from their base namespace. If you prefer the old way, I can live with that though.